mbzirc_ign/launch/competition.launch.py

```diff
@@ -28,16 +28,17 @@ def launch(context, *args, **kwargs):
     sim_mode = LaunchConfiguration('sim_mode').perform(context)
     bridge_competition_topics = LaunchConfiguration('bridge_competition_topics').perform(context)
     robot = LaunchConfiguration('robot').perform(context)
+    headless = LaunchConfiguration('headless').perform(context)
```

Comment:
I think doing this can help remove all the string comparisons below.  Same for bridge_competition_topics.

```suggestion
    headless = bool(LaunchConfiguration('headless').perform(context))
```

Reply:
hmm this gives me:

```
>>> bool('False')
True
```

Right now the user specifies `headless:=False bridge_competition_topics:=False`
So for the above to work, we'll need to change the launch arg to integer?

Reply:
ok done. https://github.com/osrf/mbzirc/pull/149/commits/fa653dbb78d88eb551384c196ca30ccc2eb590c9